protocols/routing.rst

```diff
@@ -100,8 +100,8 @@ These summary advertisements are flooded through the backbone area attached to r
 
 On the other hand, consider the prefixes `2001:db8:aaaa:0000::/64` and `2001:db8:aaaa:0001::/64` that are inside `area 1`. Router `RA` is the only area border router that is attached to this area. This router can create two different network summary advertisements :
 
- - `2001:db8:aaaa:0000::/64` at a distance of `1` and `2001:db8:aaaa:0001::/64` at a distance of `2` from `RA`
- - `2001:db8:aaaa:0000::/63` at a distance of `2` from `RA`
+ - `2001:db8:aaaa:0001::/64` at a distance of `1` and `2001:db8:aaaa:0000::/64` at a distance of `2` from `RA`
+ - `2001:db8:aaaa:0000::/64` at a distance of `2` from `RA`
```

Comment:
This change isn't correct. The text mentions that RA may advertise precise and less-specific prefixes, hence the `/63`. There is no change in the first bullet.

Reply:
Ah yes indeed, you're right for the first bullet.